README.md

```diff
@@ -120,3 +120,6 @@ public release of the library.
 [issues-closed-url]: http://www.issuestats.com/github/aarnott/pinvoke
 
 [PInvokeCoverageReport]: https://ci.appveyor.com/api/projects/aarnott/pinvoke/artifacts/coverage.html?branch=master
+
+## What if I need custom `uint` value not provided in `enum`?
```

Comment:
Can you move this to the bottom of the `Usage` section and use a heading style that is the next one smaller than `Usage`?